### PR TITLE
Feature/zoomableherosheet

### PR DESCRIPTION
**Pull request by manuelstengelberger**
Restructured hero sheet options so that they appear in one fixed container and added a zoom option with the following fixed zoomlevels: 100%, 125%, 150%, 175%, 200%
Setting a zoom level doesn't affect the "save to pdf" function

When the sheet was wider than the scroll view, the horizontal scrollbar wasn't visible because it was out of the scroll view's viewport. Fixed that, too.

### CODE REVIEW

**Comment by manuelstengelberger** [comment]
PS: fixed lint errors. the remaining warnings are from other parts of the code

**Comment by manuelstengelberger** [comment]
Switched zoomFactor type to number instead of string so the class for the DOM element is now generated like this:
`Just (`zoom-${zoomFactor}`)`
I used 100, 125 etc instead of 1, 1.25 etc so I don't have to recalculate it before generating the css class

i18n entry will probably follow this evening

**Comment by manuelstengelberger** [comment]
i10n support implemented
see pr in optolith-data

File: app/Schema/Config/Config.d.ts [hunk 1]
```diff
@@ -22,6 +22,7 @@ export interface RawConfig {
   equipmentGroupVisibilityFilter: number
   sheetCheckAttributeValueVisibility?: boolean
   sheetUseParchment?: boolean
+  sheetZoomFactor?: number | Nothing
```
Comment (elyukai) [inline app/Schema/Config/Config.d.ts:25]: It cannot be `Nothing`, only `undefined` or `number`.

**Pull request merged**